code/__DEFINES/vote.dm

```diff
@@ -0,0 +1,7 @@
+#define PLURALITY_VOTING 0
+#define APPROVAL_VOTING 1
+
+GLOBAL_LIST_INIT(vote_type_names,list(\
```

Comment:
why is this a global list

Reply:
Shit, I mindlessly copied that from my Cit implementation where (IIRC) I used it more than once. Yeah, I'll just make it in the actual proc.